### PR TITLE
[ci] move window serve tests to rayci

### PR DESCRIPTION
**Pull request by can-anyscale**
Move window serve tests to rayci. Finished within 60 minutes and use 2 shards.

Test:
- CI

### CODE REVIEW

File: .buildkite/serve.rayci.yml [hunk 1]
```diff
@@ -13,6 +13,16 @@ steps:
   - name: servepython311build
     wanda: ci/docker/servepython311.build.wanda.yaml
 
+  - name: minbuild-serve
```
Comment (can-anyscale) [inline .buildkite/serve.rayci.yml:16]: move this while i'm here so the build are in one place

File: python/ray/serve/tests/BUILD [hunk 2]
```diff
@@ -30,9 +30,8 @@ py_test_module_list(
     "test_cluster_node_info_cache.py",
     "test_long_poll.py",
     "test_util.py",
-    "test_cross_language.py",
     "test_proxy.py",
-    "test_deployment_graph_ingress_deployment.py",
+    "test_deployment_graph_ingress.py",
```
Comment (can-anyscale) [inline python/ray/serve/tests/BUILD:34]: The name of the test is longer than windows limit (as we change bazel output from c/tmp to c/raytmp). Shorten it a bit.

Comment (edoakes) [reply python/ray/serve/tests/BUILD:34]: @zcin can we delete this test?

**Review by edoakes — APPROVED** [review]

File: python/ray/serve/tests/BUILD (hunk 2 above)
Comment (can-anyscale) [reply python/ray/serve/tests/BUILD:34]: feel free to let me know if you want to delete this one too!

**Pull request merged**